game/tests.py

```
@@ -201,11 +199,134 @@ def test_duplicate_email_registration_fails(self):
+        
+        # Register again with same details to trigger re-verification
+        payload = {
+            'username': 'reverify',
+            'email': 'reverify@example.com',
+            'password1': 'StrongPass123!',
+            'password2': 'StrongPass123!',
+        }
+        
+        # Capture the OTP printed in debug mode (mocked settings)
+        with override_settings(DEBUG=True, EMAIL_HOST_USER='', EMAIL_HOST_PASSWORD=''):
+            with mock.patch('builtins.print') as mock_print:
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)